data/applications.yml

```diff
@@ -347,3 +337,18 @@
   team: "#publishing-frontend"
   product_manager: "@humin_miah"
   component_guide_url: https://govuk-static.herokuapp.com/component-guide
+
+# Transition
+
+- github_repo_name: bouncer
+  type: Transition apps
+  product_manager: "@lukemalcher"
+  team: "#taxonomy"
+
+- github_repo_name: transition
+  type: Transition apps
+  product_manager: "@lukemalcher"
+  team: "#taxonomy"
+
+- github_repo_name: side-by-side-browser
```

Comment:
Most of the links on the application page (Sentry / Puppet configuration / Deploy scripts / Deployment dashboard) won't work for this app because it's not hosted on our own infrastructure (it's on Heroku). Perhaps move it to the `Supporting transition` in dashboard.yml?

Reply:
Makes sense.  Rebased.